docs/examples/frameworks/jax/jax-multigpu_example.ipynb

```diff
@@ -7,27 +7,58 @@
    "source": [
```

Comment:
<blockquote>Here we show how to run training from <a href="https://github.com/NVIDIA/DALI/raw/b5fade136427602a13eb071d7c560a31cea2df47/docs/examples/frameworks/jax/jax-basic_example.ipynb" target="_blank">Training neural network with DALI and JAX</a> on multiple GPUs. If you haven't already done so it is best to start with single GPU example to better understand following content.  </blockquote><p>I may be picky or just drowsy, but I read <code>If you haven't already done so</code> as if referring to training NN on multiple GPUs. How about something like:</p><blockquote>Here we show how to run training from <a href="https://github.com/NVIDIA/DALI/raw/b5fade136427602a13eb071d7c560a31cea2df47/docs/examples/frameworks/jax/jax-basic_example.ipynb" target="_blank">Training neural network with DALI and JAX</a> on multiple GPUs. It is best to start with single GPU example, if you haven't already done so.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done. Changed the whole thing</p>